sparrow-py/src/expr.rs

```diff
@@ -31,9 +36,16 @@ impl Expr {
         let args: Vec<_> = args
             .into_iter()
             .map(|arg| {
-                arg.into_ast_dfg_ref(&mut rust_session)
-                    // DO NOT SUBMIT: Better error handling.
-                    .map_err(|_| PyRuntimeError::new_err("unable to create literal"))
+                match arg {
+                    None => rust_session
+                        .add_literal(Literal::Null)
+                        .map_err(|_| PyRuntimeError::new_err("unable to create null literal")),
+                    Some(arg) => {
+                        arg.into_ast_dfg_ref(&mut rust_session)
+                            // DO NOT SUBMIT: Better error handling.
```

Comment:
blocker?

Reply:
not really -- definitely need to really improve the error story, but TBD